lib/perl/Genome/Model/SomaticVariation/Command/Loh.pm

```diff
@@ -130,6 +130,12 @@ sub should_skip_run {
         return 1;
     }
 
+    my $normal_build = $build->normal_build;
+    unless($normal_build->can('snv_detection_strategy') and $normal_build->snv_detection_strategy) {
```

Comment:
This probably won't work since normal_build is ref-align build that doesn't have snv_detection_strategy as property (unlike som-var build).  Instead, $normal_model should work.